src/GraphControl/Control/Grapher.cpp

```diff
@@ -404,6 +408,7 @@ namespace GraphControl
                         m_renderMain->Graph = m_graph;
 
                         UpdateVariables();
+                        UpdateKeyGraphFeatures();
```

Comment:
Could we delay this until the user clicks the button? I know we can't detect the specific equation yet, but it would be good to minimize the performance hit when we initially display the graph as much as possible.

Reply:
Yes, but it would be best to do this after the code is refactored to use an equation model. I haven't noticed much of a performance hit in release with the code this way.